paper.tex

```diff
@@ -170,14 +170,14 @@ \subsubsection{Existing RSE efforts}
 
 \todo{FLO+PMS: They are not an example for an institutional RSE structure. We can scrap this, paragraph starting here} The Carpentries\footnote{\url{https://carpentries.org}}\autocite{Wilson2006} exemplify a similar success story\footnote{Carpentries25 Testimonial Series: \url{https://carpentries.org/blog/tag/carpentries25/}}.
 \todo{Assertion-citation mismatch.} Requests or suggestions for even more training show the need for such services\footnote{Carpentries Incubator and Carpentries Lab: \url{https://carpentries.org/lesson-development/community-lessons/}}.
-RSE services which benefit all disciplines/departments may represent a unique selling point for organisations competing for the brightest minds, see the examples from leading German universities above.\todo{PMS: Scrap whole paragraph except for first sentence?}
+RSE services which benefit all disciplines/departments may represent a unique selling point for organisations competing for the brightest minds, see the examples from leading German research performing organisations above.\todo{PMS: Scrap whole paragraph except for first sentence?}
```

Comment:
So, is "research performing organisations" the most general term? It sounds a bit clumsy to me.  Can't we just to "research organisations" and explain above what's meant by that? It feels like we're trying to be too broad/correct here.

Reply:
I was wondering about that, google suggested that's a term but maybe we can get away with research organisations?

Reply:
It has a defined meaning in the science management domain, RPOs and RFOs(Research Funding Organizations). It'salso  in line with Position-Paper-1 https://f1000research.com/articles/9-295